wayang-platforms/wayang-java/src/main/resources/wayang-java-defaults.properties

```diff
@@ -341,69 +341,89 @@ wayang.java.textfilesource.load.main = {\
   "p":0.9\
 }
 
-
```

Comment:
It seems some changes from the previous PR are not included here. Maybe try to pull the newest changes